### DISCUSSION TITLE
How to upload a model artifact using REST API ?

### DISCUSSION THREAD

**Discussion by guzman109** [mlflow/mlflow#11536] [Q&A]
I see the Python sdk has mlflow.register_model or the log_model methods using a backend. 

Can we upload a model artifact using the rest apis? Does anyone have an example or tutorial on how to do so? I couldn't find one online.

**Comment by rkube** [comment]
This would be a fantastic feature. I’m doing some model training in Julia and while the rest API works great, I’m missing artifacts.

**Reply by guzman109** [reply]
If you trace it, you can find the api endpoint. It's been awhile, but I was able to do it.

**Reply by rkube** [reply]
How would I trace this? Using wireshark?

**Reply by guzman109** [reply]
No, I just used debug or dev logging when I used the SDK and I saw the api calls in the logs.
